lib/handlers/health_check_handler.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+
+import 'package:another_flushbar/flushbar.dart';
+import 'package:breez_sdk/bridge_generated.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/health_check/health_check_bloc.dart';
+import 'package:c_breez/handlers/handler.dart';
+import 'package:c_breez/handlers/handler_context_provider.dart';
+import 'package:c_breez/widgets/flushbar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger("HealthCheckHandler");
+
+class HealthCheckHandler extends Handler {
+  StreamSubscription<HealthCheckStatus>? _listener;
+
+  @override
+  void init(HandlerContextProvider<StatefulWidget> contextProvider) {
+    super.init(contextProvider);
+    final context = contextProvider.getBuildContext();
+    if (context == null) {
+      _log.info("HealthCheckHandler: context is null");
+      return;
+    }
+
+    final texts = context.texts();
+    final themeData = Theme.of(context);
+    final bloc = context.read<HealthCheckBloc>();
+
+    _listener?.cancel();
+    _listener = bloc.stream.listen((event) {
+      if (event == HealthCheckStatus.Maintenance || event == HealthCheckStatus.ServiceDisruption) {
```

Comment:
Shouldn't we also remove the flushbar when the helth check is normal again?

Reply:
good catch; fixed.